codecov.yml

```diff
@@ -82,3 +82,4 @@ ignore:
   - "precompiles/**/*.sol"
   - "precompiles/prototype/IPrototype.go"
   - "precompiles/staking/IStaking.go"
+  - "precompiles/bank/IBank.go"
```

Comment:
if we could somehow make all the generated code be like `IBank.gen.go` we could just ignore `*.gen.go`

Reply:
Done in 67ad4faac4bce888d9cb89af9b5308f53cccc76a